src/sass_context.cpp

```diff
@@ -112,7 +112,7 @@ extern "C" {
       c_ctx->error_file = sass_strdup(e.pstate.path);
       c_ctx->error_line = e.pstate.line+1;
       c_ctx->error_column = e.pstate.column+1;
-      c_ctx->error_src = e.pstate.src;
+      c_ctx->error_src = sass_strdup(e.pstate.src);
```

Comment:
I don't feel too confortable to copy the whole source if avoidable. We should see if we can delay the de-allocation then. I do see the point that it's propably only affecting error cases. But I would like to check first if we can delay it.